test/recipes/80-test_ssl_old.t

```diff
@@ -527,11 +527,14 @@ sub testssl {
                 if $no_tls1 || $provider eq "fips";
 
         SKIP: {
-            skip "skipping anonymous DH tests", 1
+            skip "skipping anonymous DH tests", 2
                 if ($no_dh);
 
             ok(run(test([@ssltest, "-v", "-bio_pair", "-tls1", "-cipher", "ADH", "-dhe1024dsa", "-num", "10", "-f", "-time"])),
                'test tlsv1 with 1024bit anonymous DH, multiple handshakes');
+
```

Comment:
Does this need its own skip section since it is both a if (no_dh) AND (no_psk)

Reply:
Yes it does as I later discovered...
The 1.1.1 PR is breaking for other reasons...